CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -39,7 +39,7 @@ public class StudentsController : ControllerBase
         /// <summary>
         /// Addition of new student
         /// </summary>
-        /// <response code="200">Successful passing of account into student</response>
+        /// <response code="200">Successful assigning of account into student</response>
```

Comment:
* Wrong use of preposition here ('into'). It's _for_ student, not _into_ student.
* And a wrong message. Essentially, what this endpoint doing is assigning a role for a given account.

So it should be something like this: "New student is successfully added"

```suggestion
        /// <response code="200">New student is successfully added.</response>
```